copying.md

```diff
@@ -108,6 +108,7 @@ _the openage authors_ are:
 | Julian Guillotel            | arialwhite                  | julian dawt gullotel à gmail dawt com             |
 | Arne Sellmann               | PythonicChemist             | arne dawt sellmann à gmx dawt de                  |
 | Rafael X. Morales Georgi    | chocoladisco                | chocoladisco á gmail dawt com                     |
+| Marcel Schneider            | schnema123                  | marcelschneider5 á outlook dawt de                |
```

Comment:
Replace `á` with `à` (similar to line 109 above).
You can optionally fix the last-but-one line also.